actions.yaml

```diff
@@ -1,16 +1,2 @@
 # Copyright 2021 Unicorn
```

Comment:
If the charm does not have any actions, this file can be removed completely.

Reply:
Good point, I don't think there will be any action in the near future.